File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.js
function solution(numbers, direction) {
    const answer = [];
    
    if(direction === "right") {
        for(let i = 0; i < numbers.length - 1; i++) {
            answer[i+1] = numbers[i];
        }
        answer[0] = numbers[numbers.length - 1];
    } else {
        for(let i = 0; i < numbers.length - 1; i++) {
            answer[i] = numbers[i+1];
        }
        answer[numbers.length - 1] = numbers[0];
    }
    
    return answer;
}

File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.go
func solution(numbers []int, direction string) []int {
    answer := make([]int, len(numbers))
        
    if direction == "right" {
        for i := 0; i < len(numbers)-1; i++ {
            answer[i+1] = numbers[i]
        }

        answer[0] = numbers[len(numbers)-1]
    } else {
        for i := 0; i < len(numbers)-1; i++ {
            answer[i] = numbers[i+1]
        }

        answer[len(numbers)-1] = numbers[0]
    }

    return answer
}